src/components/jsmd-editor.jsx

```diff
@@ -141,6 +152,7 @@ class JsmdEditorUnconnected extends React.Component {
   };
 
   render() {
+    // FIXME: restore autocomplete to working order
```

Comment:
@hamilton i added a fixme for the commented out block of code having to do with autocomplete. if we stick with codemirror, this will be helpful in reimplementing that; if we switch, it's irrelevant anyway